lstchain/calib/camera/pulse_time_correction.py

```diff
@@ -91,6 +90,35 @@ def get_corr_pulse(self, event, pulse):
                                 self.n_capacitors)
         return pulse_corr
 
+    def get_pulse_time_corrections(self, event):
```

Comment:
Isn't this the same code that is now available in `ctapipe_io_lst`? Can't we just import it here if it is needed?

Reply:
E.g. this is the same as `ctapipe_io_lst.calibration.calc_drs4_time_correction_both_gains`, right?

Reply:
yes, this code is not more used and  can be removed. As said above, for the moment I did not remove the code in lstchain but if we agree we can do it.

Reply:
I think we should to make sure we move to the new code everywhere where needed and be safe that nothing is still using the old code.